docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -37,27 +37,34 @@ In Iris, lazy data is provided as a
 A dask array also has a shape and data type
 but typically the dask array's data points are not loaded into memory.
 Instead the data points are stored on disk and only loaded into memory in
-small chunks when absolutely necessary.
+small chunks when absolutely necessary (see the section :ref:`when_real_data`
```

Comment:
@dkillick This addresses @bjlittle's comment about 'absolutely necessary', I am still undecided about whether to include a link to the dask docs regarding chunks.

It is a little ambiguous whether the phrase 'small chunks' in this paragraph refers to dask chunks specifically, or just some unofficial chunks of stuff.  I assume that by not adding a link to the docs, you have decided that in this case, you are using the term unofficially.  Are we all happy with this?